src/airport.js

```diff
@@ -1,26 +1,28 @@
 
 class Airport {
   airportPlanes;
+  capacity;
   constructor (typeOfTransport, capacity=20, airportPlanes=[]){
```

Comment:
I think that there is a better way to create this constructor but I'm not sure how. In this case, for instance, if I want to initialise an airport object with several planes but without changing the default capacity, I still need to enter a capacity as an argument. Maybe changing the order would help: constructor (typeOfTransport, airportPlanes=[], capacity=20){